server/front/src/index.ts

```diff
@@ -274,32 +274,80 @@ export function start (
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
   app.get('/files/*', filesHandler)
 
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  // app.post('/files', async (req, res) => {
+  //   const file = req.files?.file as UploadedFile
+  //
+  //   if (file === undefined) {
+  //     res.status(400).send()
+  //     return
+  //   }
+  //
+  //   const authHeader = req.headers.authorization
+  //   if (authHeader === undefined) {
+  //     res.status(403).send()
+  //     return
+  //   }
+  //
+  //   try {
+  //     const token = authHeader.split(' ')[1]
+  //     const payload = decodeToken(token)
+  //     const uuid = await minioUpload(config.minio, payload.workspace, file)
+  //
+  //     res.status(200).send(uuid)
+  //   } catch (error) {
+  //     console.log(error)
+  //     res.status(500).send()
+  //   }
+  // })
+
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
   app.post('/files', async (req, res) => {
-    const file = req.files?.file as UploadedFile
+    const MAX_FILE_SIZE = 10 * 1024 * 1024; // For example, 10 MB
```

Comment:
Parameters like this ones should be possible to configure, size limits for purpose are configurable via external service, like Kubernetes Ingress for us.

For mimetypes is a good idea, but also could be good to have it to be a bit more rich and configurable outside of front pod.